test/JDBC/expected/BABEL-3820-vu-prepare.out

```diff
@@ -0,0 +1,95 @@
+-- Expect error
+CREATE PROCEDURE openjson_3820_p1
+AS
+BEGIN
+    SELECT * FROM OPENJSON('{}') WITH(field int 'strict$.field')
```

Comment:
Can you give a comment in the test case indicate what's the purpose of the test case and what's the expected output.